.github/workflows/build-image-main.yml

```
@@ -2,24 +2,34 @@
 
 jobs:
+  fetch-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Fetch Latest Release Tag
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:34: Double quote to prevent globbing and word splitting

(shellcheck)